libraries/botframework-connector/package.json

```diff
@@ -27,21 +27,21 @@
     }
   },
   "dependencies": {
-    "@azure/identity": "2.0.0-beta.6",
-    "@azure/ms-rest-js": "1.9.1",
-    "@types/jsonwebtoken": "7.2.8",
-    "@types/node": "^10.17.27",
+    "@azure/identity": "^2.0.4",
+    "@azure/ms-rest-js": "^2.6.1",
     "adal-node": "0.2.3",
     "axios": "^0.25.0",
     "base64url": "^3.0.0",
     "botbuilder-stdlib": "4.1.6",
     "botframework-schema": "4.1.6",
     "cross-fetch": "^3.0.5",
-    "jsonwebtoken": "8.0.1",
+    "jsonwebtoken": "^8.0.1",
```

Comment:
From what i've seen in the yarn.lock this is resolving the `8.5.1` version, shouldn't we also update this to ^8.5.1

Reply:
Feedback applied. Thanks!